### PR TITLE
Use primary screen instead of current screen

### PR DESCRIPTION
**Pull request by tma02**

### CODE REVIEW

**Comment by sindresorhus** [comment]
Can you link to any documentation about the first screen being the primary screen?

File: wallpaper.m [hunk 1]
```diff
@@ -12,7 +12,8 @@ int main() {
 	@autoreleasepool {
 		NSWorkspace *sw = [NSWorkspace sharedWorkspace];
 		NSArray *args = [NSProcessInfo processInfo].arguments;
-		NSScreen *screen = [NSScreen mainScreen];
+        NSArray<NSScreen *> *screens = [NSScreen screens];
+        NSScreen *screen = screens[0];
```
Comment (sindresorhus) [inline wallpaper.m]: Incorrect indentation.

**Comment by tma02** [comment]
Sure thing; https://developer.apple.com/library/mac/documentation/Cocoa/Reference/ApplicationKit/Classes/NSScreen_Class/#//apple_ref/occ/clm/NSScreen/screens
>The screen at index 0 in the returned array corresponds to the primary screen of the user’s system. This is the screen that contains the menu bar and whose origin is at the point (0, 0). In the case of mirroring, the first screen is the largest drawable display; if all screens are the same size, it is the screen with the highest pixel depth. This primary screen may not be the same as the one returned by the mainScreen method, which returns the screen with the active window.

File: wallpaper.m (hunk 1 above)
Comment (sindresorhus) [inline wallpaper.m]: Couldn't you just do `NSScreen *screen = [NSScreen screens].firstObject;`?

Comment (tma02) [inline wallpaper.m]: Sorry, I'm not an Objective-C developer

**Pull request closed without merging**